engines/bastion_katello/app/assets/javascripts/bastion_katello/products/details/views/product-repositories.html

```diff
@@ -149,6 +149,15 @@
                   </span>
                 </div>
               </span>
+
+              <span ng-show="repository.content_type == 'ostree'">
+                <div>
+                  <span translate>
+                    {{ repository.content_counts.ostree_branch || 0 }} OSTree Units
```

Comment:
Although we do not appear to be handling that in this area for other content types.